load/loader.go

```diff
@@ -191,6 +216,16 @@ func (l *CommonBenchmarkRunner) RunBenchmark(b targets.Benchmark) {
 		capacity = l.Workers
 	}
 
+	l.metricCounter = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "tsbs_load_metric_count",
+		Help: "TSBS: The total number of metrics inserted",
+	})
+
+	l.rowCounter = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "tsbs_load_rows_count",
+		Help: "TSBS: The total number of rows ingested",
+	})
```

Comment:
If those metrics are shipped to prometheus via pushgateway, then prometheus internal staleness mechanism won't kick in and we can accidentally mix data. It would be nice to add metrics on `benchmark_start_timestamp` and `benchmark_duration`

Reply:
Maybe it would be also nice to add a random identifier as a metric label to allow identifying particular runs.

Reply:
Great point @paulfantom ! I'll add it. I think the unique identifier is also a key point to identify. Especially when the benchmark fail for some reason, so maybe we can also report it in somehow 🤔